src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/AbstractTask.groovy

```diff
@@ -70,4 +101,201 @@ abstract class AbstractTask extends DefaultTask {
         return (project.liberty.userDir == null) ? new File(installDir, 'usr') : new File(project.liberty.userDir)
     }
 
+    //Sets directories for the various configuration files
+    protected void initializeConfigVariables(){
+        if(project.liberty.configDirectory != null && project.liberty.configDirectory.length() != 0){
+            configDirectory = new File(project.liberty.configDirectory)
+        }
+
+        if(project.liberty.configFile != null && project.liberty.configFile.length() != 0){
+            configFile = new File(project.liberty.configFile)
+        }
+        else{
+            configFile = new File("/src/main/liberty/config/server.xml")
```

Comment:
Instead of setting defaults here we can handle default values in LibertyExtention.groovy and remove this initialization. We also can remove the add properties and just reference project properties using `liberty.project.whatever` syntax.